forest/shared/src/cli/mod.rs

```diff
@@ -204,7 +280,23 @@ impl ConfigPath {
     }
 }
 
-fn find_config_path(opts: &CliOpts) -> Option<ConfigPath> {
+fn find_cli_config_path(opts: &CliOpts) -> Option<ConfigPath> {
+    if let Some(s) = &opts.config {
+        return Some(ConfigPath::Cli(PathBuf::from(s)));
+    }
+    if let Ok(s) = std::env::var("FOREST_CONFIG_PATH") {
+        return Some(ConfigPath::Env(PathBuf::from(s)));
+    }
+    if let Some(dir) = ProjectDirs::from("com", "ChainSafe", "Forest") {
+        let path = dir.config_dir().join("config.toml");
+        if path.exists() {
+            return Some(ConfigPath::Project(path));
+        }
+    }
+    None
+}
```

Comment:
`find_cli_config_path` and `find_daemon_config_path` should be merged. Or refactored such that they share their logic.